### ISSUE TITLE
Add a little documentation for A100 use

### ISSUE THREAD

**Issue by shadowlocked** [ShivamShrirao/diffusers#229]
**Is your feature request related to a problem? Please describe.**
Since Torch2, the Shivam Colab fails, at least for me, 3/4 of the time on a basic T4. The Colab was established as a tightly-optimized script with no margin for error, and the advent of Torch 2 has (in my experience) made a successful training on a T4, on this Colab, now relatively precarious. 

For those of us, like me, who are solving the problem by spending out on an A100 instance, there is no comprehensive documentation that explains how to set the script up for an A100 instead of a T4.

**Describe the solution you'd like**
I would like the Colab to contain 1 paragraph of extra advice for use on an A100, which at least mentions useful but unreferenced flags such as '--not_cache_latents', and any other flags that would help the user to maximize use of the A100, but which are not actually present currently, either as commented-out code or in the help text.

**Describe alternatives you've considered**
I have trawled Reddit, Discord, Google and even PMs with Shivam, in search of a comprehensive guide to rejigging the settings for an A100, but the advice is scrappy, and the best place for a definitive para of advice is in the Colab itself.

It should also be mentioned that actions such as not using FP16 (i.e., setting the flag to 'no') will likely slow down training considerably, as might deleting 8-Bit Adam flag from the training parameters. 

The help text could put forward the pros and cons of these changes.

**Comment by Leomn1234** [comment]
Probably some of these are use case, so depends on the training task at hand I would imagine. I use the A100 on Google colab. If you are training on houses, landscapes, etc. 8-Bit is prob fine, but for more complex structures and humans, etc. Removing the flag will work better. The only thing I do differently is remove the 8-Bit Adam flag, I still use fp16 as I've tried both and not seen a difference, it takes twice as long. What I'm getting at is I doubt there is much difference, there's literally just a couple of things you can do to improve performance, regarding the flags. Many other things can be changed that don't involve them, such as the noise scheduler, dropout, unet cross attention dim, leaning rate, weight decay, adam betas and the optimizer

**Comment by shadowlocked** [comment]
Getting this error today:

```
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 224.5/224.5 kB 6.7 MB/s eta 0:00:00
  Building wheel for diffusers (pyproject.toml) ... done
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 63.3/63.3 MB 12.1 MB/s eta 0:00:00
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 215.3/215.3 kB 5.8 MB/s eta 0:00:00
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 7.0/7.0 MB 68.6 MB/s eta 0:00:00
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 53.1/53.1 kB 6.6 MB/s eta 0:00:00
     ━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━ 34.1/62.5 MB 93.0 MB/s eta 0:00:01
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    bitsandbytes==0.35.0 from https://files.pythonhosted.org/packages/aa/f1/6ff7784b946672e59e84470a1ee31eba48a2f03dad85eee7f6bdca6e0a20/bitsandbytes-0.35.0-py3-none-any.whl:
        Expected sha256 3ba4046114b7bc66284baa8dac6bb00e2e57881a31467e5e98c34de27b0ca13c
             Got        88ce0633c6bfa1b292cc0cec7218f2dc122ae6ac4c25555c966ddb0761d6c372
```
